assets/js/components/ExecutionResults/CheckResultOutline.jsx

```diff
@@ -1,10 +1,8 @@
 import React from 'react';
-import { EXPECT, EXPECT_SAME } from '@lib/model';
+import { EXPECT, EXPECT_SAME, TARGET_CLUSTER, TARGET_HOST } from '@lib/model';
+import { useNavigate } from 'react-router-dom';
```

Comment:
Could you invert these?

```suggestion
import { useNavigate } from 'react-router-dom';

import { EXPECT, EXPECT_SAME, TARGET_CLUSTER, TARGET_HOST } from '@lib/model';
```